optuna/visualization/_intermediate_values.py

```diff
@@ -113,12 +123,16 @@ def _get_intermediate_plot(info: _IntermediatePlotInfo) -> "go.Figure":
     if len(trial_infos) == 0:
         return go.Figure(data=[], layout=layout)
 
+    default_marker = {"maxdisplayed": 10}
+
     traces = [
         go.Scatter(
             x=tuple((x for x, _ in tinfo.sorted_intermediate_values)),
             y=tuple((y for _, y in tinfo.sorted_intermediate_values)),
             mode="lines+markers",
-            marker={"maxdisplayed": 10},
+            marker=default_marker
+            if tinfo.feasible
+            else default_marker.update({"color": "#CCCCCC"}),  # type: ignore[dict-item]
```

Comment:
Sorry for the confusion but `dict.update()` returns `None` like `list.sort()` Please change this part like this.
```suggestion
            else {**default_marker, "color": "#CCCCCC"},
```

Reply:
Of course - silly mistake. Thank you :)